demo-nav.sh

```diff
@@ -8,7 +8,7 @@
 # TYPE_SPEED=40
 #
 # If empty next command will be shown only after enter (Default: false).
-IMMEDIATE_REVEAL=false
+# IMMEDIATE_REVEAL=false
```

Comment:
This is to make script much easier. When command is executed we need to hit additional `enter` to reveal it - it will help us to show things better